Tools/scripts/build_options.py

```diff
@@ -31,6 +31,16 @@ def config_option(self):
 # list of build options to offer NOTE: the dependencies must be
 # written as a single string with commas and no spaces,
 # eg. 'dependency1,dependency2'
+
+
+# Feature(group, name, dependency, description, default, callback)
+#
+# group        : Logical group/category used to organise related features
+# name         : Feature name within the group
```

Comment:
The name is a unique identifier for the feature. It is not unique just _within_ the group but rather globally unique. 

It should also be ma 30 characters long.